app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/StylizedMessage.kt

```diff
@@ -28,58 +37,56 @@ fun StylizedMessage(
   isUserMessage: Boolean,
 ) {
   val textToRender = message?.replace("\"", "") ?: stringResource(fallbackResId)
-  val segments = remember(textToRender) { parseStylizedText(textToRender) }
+  val paragraphs = remember(textToRender) { textToRender.split("\n\n") }
 
   val textColor = if (isUserMessage) Palette.Black else Palette.White
 
-  // Build annotated string with URL annotations
-  val annotatedString = remember(segments) {
-    AnnotatedString.Builder().apply {
-      segments.forEach { segment ->
-        when (segment) {
-          is TextSegment.Plain -> append(segment.text)
-          is TextSegment.Bold -> withStyle(SpanStyle(fontWeight = FontWeight.Bold)) {
-            append(segment.text)
+  Column {
+    paragraphs.forEachIndexed { pIndex, paragraphText ->
+      // Parse bolds/links within this paragraph
+      val segments = parseStylizedText(paragraphText)
```

Comment:
Small performance comment: could have a val outside with the parselized paragraphtext, to avoid parsing them again in case of recomposition. This allows caching those parsed paragraph texts, and not having to parse it again everytime it gets recomposed.